crates/common_utils/src/id_type.rs

```diff
@@ -50,6 +52,13 @@ fn get_invalid_input_character(input_string: Cow<'static, str>) -> Option<char>
 /// A type for alphanumeric ids
 pub(crate) struct AlphaNumericId(String);
 
+impl Default for AlphaNumericId {
```

Comment:
Let's avoid implementing / deriving `Default` on `AlphaNumericId` and `LengthId`.

Reply:
In order to derive Default on CellId, we need to be able to construct the internal types with some default value.
Since the internal types are defined as private, we can't use them directly, unless we use some conversion like `LengthId::from_alphanumeric_id` which performs checks and returns a result @SanchithHegde